spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -147,6 +147,7 @@
 		<dependency>
 			<groupId>org.mariadb.jdbc</groupId>
 			<artifactId>mariadb-java-client</artifactId>
+            <version>${mariadb-java-client.version}</version>
```

Comment:
Version would already come from parent pom's dep management?

Reply:
I just try to be explicit, can test.

Reply:
seems to work w/o it, so removing it.